data/defaults.csv

```diff
@@ -0,0 +1,6 @@
+env_setup_commands,pip install --upgrade pip<19
+env_setup_commands,poetry install
+env_setup_commands,pre-commit install
```

Comment:
Additionally, I don't know that we should assume `pre-commit` is already installed globally so would suggest `poetry run pre-commit install`

Reply:
see above re: poetry